Content.Client/Cargo/UI/CargoBountyMenu.xaml.cs

```diff
@@ -17,8 +17,11 @@ public CargoBountyMenu()
         RobustXamlLoader.Load(this);
     }
 
-    public void UpdateEntries(List<CargoBountyData> bounties, TimeSpan untilNextSkip)
+    public void UpdateEntries(List<CargoBountyData> bounties, List<CargoBountyHistoryData> history, TimeSpan untilNextSkip)
     {
+        MasterTabContainer.SetTabTitle(0, Loc.GetString("bounty-console-tab-available-label"));
+        MasterTabContainer.SetTabTitle(1, Loc.GetString("bounty-console-tab-history-label"));
```

Comment:
I think this would make more sense in the constructor as you don't need to set the name each time the entries are opened.